pyrefinebio/dataset.py

```diff
@@ -246,7 +246,16 @@ def download(self, path, prompt=True):
         download_url = self.download_url or self.get(self.id).download_url
 
         if not download_url:
-            raise DownloadError("dataset", "Download url not found - did you process the dataset?")
+            if self.check():
+                raise DownloadError(
+                    "Dataset",
+                    "Download url not found - did you set up and activate a Token?"
```

Comment:
This good. We can tell them how they can get the token.

`You can create and activate a new token using pyrefinebio.Token(). See documentation for advanced usage <link>`